includes/MediaWiki/ReconciliationService.php

```diff
@@ -146,9 +147,15 @@ public function getOrCreateItemByStatementUrl(
 			$base->setId( ItemId::newFromNumber( $this->idGenerator->getNewId( 'wikibase-item' ) ) );
 			$reconciliationItem = new ReconciliationItem( $base, false );
 
+			$guidGenerator = new GuidGenerator();
```

Comment:
Also, if the GUID generator is injected, we can use a mocked generator in the tests which always returns `"$id\$00000000-0000-0000-0000-000000000000"`, or something like that, to avoid including part of the actual API response in the expected / comparison value.

Reply:
Done.

Reply:
> Also, if the GUID generator is injected, we can use a mocked generator in the tests which always returns `"$id\$00000000-0000-0000-0000-000000000000"`, or something like that, to avoid including part of the actual API response in the expected / comparison value.

Wait, this message wasn't here when I was adding the 'Done' comment

Reply:
Now done I think. Please re-check :)